tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "mocked-api",
     "setupTests.ts"
   ],
+  "exclude": ["dist", "node_modules"],
```

Comment:
`stories` files are already correctly excluded [in the build ts configuration](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/5c33d7cd8c7315ac2873314a5cf220776fba4f5d/tsconfig.build.json#L20), but they shouldn't be in the "dev" tsconfig, hence the override here.